tools/nodejs_api/src_cpp/include/node_progress_bar_display.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <iostream>
```

Comment:
Is `iostream` actually used in the implementation or is it only imported for printing/debugging? If not required, it should be removed.

Reply:
Yeah it's not used, accidentally left over from copying the terminal display class